components/Player/Pages/playerMatchesColumns.jsx

```diff
@@ -26,14 +26,24 @@ export default [{
 }, {
   displayName: 'Len',
   field: 'duration',
-  width: 1.5,
+  width: 1.2,
   sortFn: true,
   displayFn: transformations.duration,
 }, {
-  displayName: 'KDA',
+  displayName: 'K',
   field: 'kills',
-  sortFn: (row) => ((row.kills + row.deaths + row.assists) / row.deaths),
-  width: 2,
+  sortFn: true,
+  width: 0.8,
   displayFn: transformations.kda,
+}, {
+  displayName: 'D',
+  field: 'deaths',
+  sortFn: true,
+  width: 0.8,
+}, {
+  displayName: 'A',
+  field: 'assists',
+  sortFn: true,
+  width: 0.8,
 },
```

Comment:
I know I made a comment about this being split, but does it remove a lot of space from the table split? Do you have a screenshot? It may be worth the loss of sortability if it means we can display our data more cleanly/fit in more stuff.